Projects/HDRPInputSystem/Packages/manifest.json

```diff
@@ -47,6 +48,7 @@
   },
   "testables": [
     "com.unity.cinemachine",
```

Comment:
still need this?

Reply:
Hmm good point. May not be needed. Will try.

Reply:
Works locally. Gonna test in Yamato but isn't needed apparently. Nice catch!